app/code/Magento/Swatches/Model/Plugin/EavAttribute.php

```diff
@@ -174,10 +174,8 @@ protected function prepareOptionIds(array $optionsArray)
     {
         if (isset($optionsArray['value']) && is_array($optionsArray['value'])) {
             foreach (array_keys($optionsArray['value']) as $optionId) {
-                if (isset($optionsArray['delete']) && isset($optionsArray['delete'][$optionId])) {
-                    if ($optionsArray['delete'][$optionId]== 1) {
-                        unset($optionsArray['value'][$optionId]);
-                    }
+                if (isset($optionsArray['delete'][$optionId]) & $optionsArray['delete'][$optionId]) {
```

Comment:
I see... I don't know a way to force push via GitHub web UI as well. Let's leave it as is then.